chain/syncer_integration_test.go

```diff
@@ -330,7 +330,7 @@ func TestLoadFork(t *testing.T) {
 	// Set up chain store to have standard chain up to dstP.link2
 	_ = requirePutBlocks(t, blockSource, dstP.link1.ToSlice()...)
 	cids2 := requirePutBlocks(t, blockSource, dstP.link2.ToSlice()...)
-	err := syncer.HandleNewTipset(ctx, cids2)
+	err := syncer.HandleNewTipset(ctx, cids2, th.RequireRandomPeerID(t))
```

Comment:
Blocking: don't add nondeterminism to tests - use a fixed ID.

I suggest just `""` or `"testpeer"` if it's not to be used?